src/sentry/api/endpoints/organization_ddm.py

```diff
@@ -75,7 +75,7 @@ def _get_metric_code_locations(
             end=end,
             organization=organization,
             projects=projects,
-        )
+        )[1]
```

Comment:
We now return a tuple, thus we want to get the result directly in the old endpoint. The endpoint will be faded out in the future, so this is not a big problem.